cmd/flags/testgrid.go

```diff
@@ -0,0 +1,93 @@
+package flags
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/sippy/pkg/sippyserver"
+)
+
+type TestGridFlags struct {
+	JobFilter               string
+	LocalData               string
+	StartDay                int
+	NumDays                 int
+	TestSuccessThreshold    float64
+	MinTestRuns             int
+	FailureClusterThreshold int
+	Dashboards              []string
+}
+
+func NewTestGridFlags() *TestGridFlags {
+	return &TestGridFlags{
+		NumDays:                 14,
+		TestSuccessThreshold:    99.99,
+		MinTestRuns:             10,
+		FailureClusterThreshold: 10,
+		StartDay:                0,
+	}
+}
+
+func (f *TestGridFlags) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&f.JobFilter, "testgrid-job-filter", f.JobFilter, "Only analyze jobs that match this regex")
+	fs.StringVar(&f.LocalData, "testgrid-local-data", f.LocalData, "Path to testgrid data from local disk")
```

Comment:
Looks like these are used in a testgrid sub-sub-command, could we removed the "testgrid-" prefix from the option names so we're less verbose?

Reply:
They also get used on the `serve` command when reading from local-data directory (not DB) so having just a "job-filter" flag is a little confusing there.  Do we perhaps just want to remove that entirely? TBH I'm not even sure if it works anymore.

Reply:
I can't think of any use left for testgrid from disk other than historical, and we've kinda got that covered now with snapshots.